### ISSUE TITLE
FR: zsh completion `ff move --to <TAB>`

### ISSUE THREAD

**Issue by ShuP1** [martinvonz/jj#2550]
**Is your feature request related to a problem? Please describe.**
With zsh completion and `git commit --fixup <TAB>`, I get a list of previous commits.
It is not the case with `ff move --to <TAB>`

This tiny issue makes `git` better for this workflow IMO vs `ff log` + copy-paste   

**Describe the solution you'd like**
Add completions listing N previous commits

**Describe alternatives you've considered**
Got back to `git`

**Comment by necauqua** [comment]
This is blocked on clap dynamic completion 🤷
Slight duplicate of #1217 - I'm sure some revset of last commit change ids will definitely be added to completion once that's possible

> \+ copy-paste

Depending on the size of the repo, the unique short prefixes of change-ids/commit-ids are often good enough to avoid mouse copy-pasting

I mean you could handwrite a completion script, like cargo does, or guys at nushell do for jj, so it's not a _hard_ hard blocker

**Comment by martinvonz** [comment]
You might also be able to use use https://github.com/rsteube/carapace-bin. It has support for jj since the latest release (0.28.3).

**Comment by avamsi** [comment]
I use the following snippet with fzf (https://github.com/avamsi/dotfiles/blob/5b94015cc062aa0c1fd8ca94e32d0aa17bd70415/.zshrc#L63-L75):

```zsh
_fzf_complete_jj() {
	_fzf_complete \
		--preview 'jj bgc show --summary {1}' \
		--preview-window wrap -- "$@" < <(
			jj bg list --revisions=interesting --template=oneline
		)
}

_fzf_complete_jj_post() {
	cut -d ' ' -f1
}
```

bg, bgc and list are my custom jj aliases (https://github.com/avamsi/dotfiles/blob/5b94015cc062aa0c1fd8ca94e32d0aa17bd70415/.jjconfig.toml#L8-L10)

![screenshot](https://github.com/martinvonz/jj/assets/9637983/075e3e42-dd4e-4498-a1d3-985fdc7ed6bb)

**Comment by necauqua** [comment]
I just realized nothing stops us from appending (a couple of most useful) manually written completions to the output of `jj util completion`

**Comment by senekor** [comment]
Note that `jj move` is deprecated, use `jj squash` instead. Both will have native completions with the incoming PR.
